lib/Bio/EnsEMBL/DataCheck/Checks/CheckSyntenySanity.pm

```diff
@@ -0,0 +1,130 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckSyntenySanity;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::Utils::SqlHelper;
```

Comment:
Yes, I think so

Reply:
I take it back, not needed! I feel like it complained once when I didn't have it... But not anymore!

Reply:
Right. I was going to check :)
Since you are not calling the package directly, you shouldn't need it (and `DBConnection::sql_helper` imports it on its own side)